packages/api/AlvTime.Business/InvoiceRate/InvoiceRateService.cs

```diff
@@ -70,24 +156,32 @@ private TaskType GetTaskType(TimeEntryWithCustomerDto entry)
 
     private decimal GetUserAvailableHours(DateTime fromDate, DateTime toDate)
     {
-        decimal redDayHours = GetAmountOfRedDayHours(fromDate, toDate);
+        decimal redDayHours = GetNonWorkingDays(fromDate, toDate).Count() * 7.5m;
```

Comment:
Vil det potensielt bli noe krøll her eller andre steder for folk som ikke jobber 100%?

Reply:
Absolutt. Jeg ser jo at en ansatt med 50% vil få feil rate. Jeg tenker at det er en PR i seg selv :+1:  Regner med at du har peiling på de prosentene og kan vise meg beste måte å hente ut stillingsprosent.